presto-native-execution/presto_cpp/main/types/PrestoToVeloxQueryPlan.cpp

```diff
@@ -2401,13 +2401,7 @@ velox::core::PlanFragment VeloxBatchQueryPlanConverter::toVeloxQueryPlan(
           "shuffle-gather",
           std::vector<core::PlanNodePtr>{partitionAndSerializeNode}));
 
-  // For presto_cpp, the last node must be the PartitionedOutputNode in order to
```

Comment:
Thanks for fixing this. This was a confusing aspect for me. Could you mention the context of why this was put in place and why it needs to be removed ? The comment in itself doesn't seem very helpful

Reply:
@mbasmanova can you update the comments in line 2354-2364 https://github.com/prestodb/presto/pull/19499/files#diff-b596594147f956ff5aadb512a14f1538d5975c551dad6ba773ac266762187734R2354-R2364

Reply:
@vermapratyush I read these comments and I do not see a need to update these. Do you have something specific in mind?

Reply:
Something that can explain why the PartitionedOutputNode needs to be removed for VeloxBatchQueryPlanConverter.